### PR TITLE
Guns & projectiles tweaks

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

- All automatic gun subtypes now have full-auto firemode.
- Rifles and L6 had their magazines capacity increased. 20 --> 30, 50 --> 80.
- Most ballistic projectiles had their damage nerfed by a tiny bit.
- Fixed meteor gun and assault rifle inhand sprite name.

## Why It's Good For The Game

- More full-auto guns in general, and it makes sense to me that "automatic" subtypes have full-auto capabilities.
- More shooting time. For L6 it makes more sense given its burst spread that can only be really used for suppressive fire.
- Slightly more time spent in a fire-fight. Makes more sense for full-auto rifle, given they usually fire in bursts.
- A fix.

## Did you test it?

Yes, everything is tested and works well.

## Authorship

Me.

## Changelog

:cl:
tweak: More guns received full-auto fire mode.
tweak: Assault rifles and machine gun had their magazine capacity increased.
tweak: Pistol and rifle bullets deal slightly less damage.
fix: Meteor gun now has proper bullet icon. Assault rifle's icon doesn't break in hands.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -192,22 +192,22 @@
...
+	damage = 35
 	armor_penetration = 35
 	penetration_modifier = 1
 
 /obj/item/projectile/bullet/rifle/t12
 	fire_sound = "gun_t12"
-	damage = 24
-	armor_penetration = 40
+	damage = 25
+	armor_penetration = 50
```
Comment (Cupax3) [inline code/modules/projectiles/projectile/bullets.dm]: I like the idea of a super-penetration weapon.
Can you decrease damage but increase penetration even more?

**Review by Cupax3 — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/projectiles/projectile/bullets.dm]: Anything less than 25 damage and it becomes weaker than most **melee** weapons, which is... yeah, not good.

Comment (Cupax3) [reply code/modules/projectiles/projectile/bullets.dm]: Ah but those melee weapons can only be used in melee range. Taking an enemy apart from range with a weapon that slowly eats away at the health while the armoured enemy has little chance of catching up is exactly what I'd go for.

Comment (EgorDinamit) [reply code/modules/projectiles/projectile/bullets.dm]: I'll give it a go then. It's not like T12 is normally spawned/used outside of admemery, at the moment.

**Comment by EgorDinamit** [comment]
@Cupax3
Updated, T12 is even funnier now.

**Review by Shadowtail117 — COMMENTED** [review]
I am not sure what to think about making bullets less powerful while giving magazines increased size. Seems like a one-way ticket to endless TDMs with bullet casings littering the floor everywhere which does not seem like what the spirit of the game is like.

**Pull request merged**